docs/src/robin_dict.md

```diff
@@ -6,7 +6,7 @@ DocTestSetup = :(using DataStructures)
 
 `RobinDict` provides a standard dictionary, conforming to the AbstractDict protocol, which uses the Robin Hood hashing algorithm with backward-shift deletion to provide improved average performance over Dict.
 
-The interface of `RobinDict` replicates that of `Dict`.
+The interface of `RobinDict` replicates that of `Dict`. This has an ordered version called `OrderedRobinDict`, which replicates the interface of `OrderedDict`.
```

Comment:
1 sentence per line gives good diffs when it is editted later.
(since a sentence is a single idea)
```suggestion
The interface of `RobinDict` replicates that of `Dict`.
This has an ordered version called `OrderedRobinDict`, which replicates the interface of `OrderedDict`.
```